Client/Frontend/Autofill/CreditCard/CreditCardSettingsView/CreditCardInputViewModel.swift

```diff
@@ -91,32 +92,32 @@ class CreditCardEditViewModel: ObservableObject {
     }
 
     @Published var expirationDate: String = "" {
-        didSet (val) {
-            guard !val.isEmpty else {
+        didSet {
+            if expirationDate.count > 4 {
```

Comment:
nit: This could be changed to a guard for early exit. I think a comment will also be helpful to understand why we check for the amount of characters here because `isExpirationValidFor` also check

Reply:
Seems like it's redundant - removed!